tests/components/matter/test_sensor.py

```diff
@@ -339,3 +339,31 @@ async def test_operational_state_sensor(
     state = hass.states.get("sensor.dishwasher_operational_state")
     assert state
     assert state.state == "extra_state"
+
+
+@pytest.mark.parametrize("node_fixture", ["yandex_smart_socket"])
+async def test_yandex_smart_socket_power_sensor(
```

Comment:
We are testing a specific feature here, using the fixture of a device.
We prefer a more descriptive name for the test, what doe sit test, not what fixture is used.